File: Coding Challenges/DailyChallenge/Python/20260123_HexValidator.py
import re

def is_valid_hex(s):
    hexRegEx = '^#([0-9a-f]{3}|[0-9a-f]{6})$'

    if re.search(hexRegEx, s, re.IGNORECASE) != None:
        return True
    else:
        return False

File: Coding Challenges/Daily Challenge/2026/January/20260123-Hex-Validator.js
function isValidHex(str) {
  const hexRegEx = /^#([0-9a-f]{3}|[0-9a-f]{6})$/gi;

  if(str.match(hexRegEx) != null) {
    return true;
  }
  else {
    return false;
  }
}